portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,25 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Home</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
+    <div class="topnav">
+      <a class="active" href="index.html">Aaron Lopes</a>
+      <a href="projects.html">projects</a>
+      <a href="socials.html">socials</a>
+      <a href="misc.html">misc</a>
+    </div>  
+    
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <br></br>
```

Comment:
This is actually invalid HTML - br is a void element, so it has no ending tag. It's just br (with angle brackets that I can't write thanks to GitHub HTML escaping.).

That said, br is rarely the right solution to what you're trying to do. It looks like you simply want to create a margin. Why not use margin in CSS?